rpc/src/v1/types/trace.rs

```diff
@@ -631,6 +632,36 @@ impl From<Executed> for TraceResults {
 	}
 }
 
+#[derive(Debug, Serialize)]
+/// A diff of some chunk of memory.
+pub struct TraceResultsWithTransHash {
+	/// The output of the call/create
+	pub output: Bytes,
+	/// The transaction trace.
+	pub trace: Vec<Trace>,
+	/// The transaction trace.
+	#[serde(rename="vmTrace")]
+	pub vm_trace: Option<VMTrace>,
+	/// The transaction trace.
+	#[serde(rename="stateDiff")]
+	pub state_diff: Option<StateDiff>,
+	/// The transaction Hash.
+	#[serde(rename="TransactionHash")]
+	pub trans_hash: Option<H256>,
```

Comment:
This should be changed to full name. And why is it Option? I think if we're using this type then the transaction hash should always be available, otherwise we'd rather use `TraceResults`?